openquake/qa_tests_data/classical/case_43/job.ini

```diff
@@ -34,7 +34,7 @@ investigation_time = 1.0
 intensity_measure_types_and_levels = {"PGA": logscale(0.005, 3.00, 20)}
 truncation_level = 3
 maximum_distance = 200
-pointsource_distance = 100
+pointsource_distance = *
```

Comment:
We will need to document this  Michele otherwise the users will be lost.

Reply:
Of course, but let us first see how this approach works out.